### DISCUSSION TITLE
Welcome to Asami Discussions!

### DISCUSSION THREAD

**Discussion by quoll** [threatgrid/asami#116] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions.
  * Share ideas.
  * Engage with others.
  * Welcome others and are open-minded. Remember that this project abides by the [Contributor Covenant](https://www.contributor-covenant.org/).

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kkharji** [comment]
Hey @quoll, I've been following the project for a long time now, and I got to say it's really awesome, interesting and enabling tool, thank you for the time any energy you've put in it so far.

I've been meaning to setup a asami and use it for storing persistent data. I think durability was added few days ago, is there an example in the code base or some sudo code.

Currently I'm trying to create a backend server where the clients can send asami/datalog queries and receive Json. 

Thanks again
